smacrobs_web/src/main/java/com/web/controller/MainController.java

```diff
@@ -65,22 +72,173 @@ public ModelAndView redirectFromFitbit(
+		}catch(Exception ex) {
+			logger.warn(ex.getMessage());
+		}
+		
+		modelAndView.addObject("fullName", fullName);
+		modelAndView.addObject("gender", gender);
+		modelAndView.addObject("age", age);
+		modelAndView.addObject("dateOfBirth", dateOfBirth);
+		modelAndView.addObject("weight", weight);
+		modelAndView.addObject("weightUnit", weightUnit);
+		modelAndView.addObject("height", height);
+		modelAndView.addObject("heightUnit", heightUnit);
+		modelAndView.addObject("avatar", avatar);
+		System.out.println(modelAndView.toString());
+		return modelAndView;
+	}
+
+	private ModelAndView getWater(ModelAndView modelAndView, WaterDetails waterDetails) {
+		String water = "0";
+		try {
+			water = waterDetails.getSummary().getWater();
+		}catch(Exception ex) {
+			logger.warn(ex.getMessage());
+		}
+		
+		modelAndView.addObject("water", Integer.parseInt(water)/1000);
+		System.out.println(modelAndView.toString());
+		return modelAndView;
+	}
+
+	private ModelAndView getFood(ModelAndView modelAndView, FoodDetails foodDetails) {
+		String sodium="0";
+		String carbs="0";
+		String fats="0";
+		String fibre="0";
+		String protein="0";
+		
+		try{
+			sodium=foodDetails.getSummary().getSodium();
+			carbs=foodDetails.getSummary().getCarbs();
+			fats=foodDetails.getSummary().getFat();
+			fibre=foodDetails.getSummary().getFiber();
+			protein=foodDetails.getSummary().getProtein();
+		}
+		catch(Exception ex){
+			logger.warn(ex.getMessage());
+		}
+		modelAndView.addObject("sodium", Integer.parseInt(sodium)/1000);
+		modelAndView.addObject("carbs", carbs);
+		modelAndView.addObject("fats", fats);
+		modelAndView.addObject("fibre", fibre);
+		modelAndView.addObject("protein", protein);
+		System.out.println(modelAndView.toString());
+		return modelAndView;
+	}
+
+	private ModelAndView getActivity(ModelAndView modelAndView,
+			ActivityDetails activityDetails) {
+		double distance=0;
+		String steps="0";
+		String floors="0";
+		try{
+			steps=activityDetails.getSummary().getSteps();
+			floors=activityDetails.getSummary().getFloors();
+			for(int i=0;i<activityDetails.getSummary().getDistances().length;i++){
+				distance+=Double.parseDouble(activityDetails.getSummary().getDistances()[i].getDistance());
+				}
+			}
+		catch(Exception ex)
+		{
+			logger.warn(ex.getMessage());
+		}
+		
+		modelAndView.addObject("distance",distance);
+		modelAndView.addObject("steps",steps);
+		modelAndView.addObject("floors",floors);
+		System.out.println(modelAndView.toString());
+		
+		return modelAndView;
+	}
+
+	private ModelAndView getHeartRate(ModelAndView modelAndView, HeartRateDetails heartRateDetails) {
+    	
+    	String heartRate = "";
+    	try{
+    		heartRate=heartRateDetails.getActivitiesHeart()[0].getValue().getRestingHeartRate(); 	
+    	}
+    	catch(Exception ex){
+    		logger.warn(ex.getMessage());
+    	}
+    	modelAndView.addObject("heartRate",heartRate);
+		return modelAndView;
+	}
+
+	private String getRedirectURL() {
```

Comment:
Adjust indentation on the whole code. I was using 4 spaces, I think you are using a tab